rust/otap-dataflow/crates/pdata/src/testing/round_trip.rs

```diff
@@ -27,6 +27,38 @@ pub fn otlp_to_otap(msg: &OtlpProtoMessage) -> OtapArrowRecords {
     }
 }
 
+/// Transcode a protocol message object to OTLP bytes.
+#[must_use]
+pub fn otlp_message_to_bytes(msg: &OtlpProtoMessage) -> OtlpProtoBytes {
+    let mut buf = Vec::new();
+    msg.encode(&mut buf).expect("encoding should not fail");
+
+    match msg {
+        OtlpProtoMessage::Logs(_) => OtlpProtoBytes::ExportLogsRequest(buf.into()),
+        OtlpProtoMessage::Metrics(_) => OtlpProtoBytes::ExportMetricsRequest(buf.into()),
+        OtlpProtoMessage::Traces(_) => OtlpProtoBytes::ExportTracesRequest(buf.into()),
+    }
+}
+
+/// Transcode a protocol message object to OTLP bytes.
```

Comment:
⛏️ copy pasta, maybe.


```suggestion
/// Transcode OTLP bytes to a protocol message object.
```